sentry/sentry.conf.example.py

```diff
@@ -291,3 +291,18 @@ def get_internal_network():
 
 # BITBUCKET_CONSUMER_KEY = 'YOUR_BITBUCKET_CONSUMER_KEY'
 # BITBUCKET_CONSUMER_SECRET = 'YOUR_BITBUCKET_CONSUMER_SECRET'
+
+##############################################
+# Suggested Fix Feature / OpenAI Integration #
+##############################################
+
+# See https://docs.sentry.io/product/issues/issue-details/ai-suggested-solution/
+# for more information about the feature. Make sure the OpenAI's privacy policy is
+# aligned with your company.
+
+# Set the OPENAI_API_KEY on the .env or .env.custom file with a valid
+# OpenAI API key to turn on the feature.
+OPENAI_API_KEY = env("OPENAI_API_KEY", "")
+
+if OPENAI_API_KEY and not OPENAI_API_KEY.isspace():
```

Comment:
Interesting. I would usually do `env("OPENAI_API_KEY", "").strip()`. ¯\\\_(ツ)\_/¯

Reply:
I don't do Python so much, so that's what I ended up doing :D

Reply:
Ah, gotcha. :)

Honestly I don't think we even really need the check. In general it would be unusual (possibly even difficult) to end up with an envvar value that was only whitespace. If it were a likely problem it should be solved in [the `env` helper function](https://github.com/getsentry/sentry/blob/23.4.0/src/sentry/conf/server.py#L67-L79) (it's not). Let's do without it.

```suggestion
if OPENAI_API_KEY:
```